SignalMessaging/contacts/OWSContactsManager.m

```diff
@@ -332,7 +332,7 @@ - (void)buildSignalAccountsAndClearStaleCache:(BOOL)shouldClearStaleCache
         NSMutableSet<NSString *> *seenRecipientIds = [NSMutableSet new];
         for (Contact *contact in contacts) {
             NSArray<SignalRecipient *> *signalRecipients = contactIdToSignalRecipientsMap[contact.uniqueId];
-            for (SignalRecipient *signalRecipient in [signalRecipients sortedArrayUsingSelector:@selector(compare:)]) {
+            for (SignalRecipient *signalRecipient in [signalRecipients sortedArrayUsingSelector:@selector((compare:))]) {
```

Comment:
Just curious: why is this necessary?

Reply:
This suppresses the "Several methods with selector 'compare:' of mismatched types are found for the @selector expression"

All kinds of objects implement compare (with params of different types), and since so many objc API's use selectors (as opposed to an actual typed method reference like you could do in swift) this is the world we live in.

This is just me annotating to the compiler that we're aware in this case.